packages/core/js-client/vite.config.ts

```diff
@@ -48,14 +50,16 @@ const config: UserConfig = {
     lib: {
       entry: "./src/index.ts",
       name: "js-client",
-      fileName: () => "index.min.js",
+      fileName: () => {
+        return "index.min.js";
+      },
       formats: ["es"],
     },
     outDir: "./dist/browser",
     rollupOptions: {
       plugins: [
         {
-          // @ts-ignore
+          // @ts-expect-error Types doesn't work here. Hack
```

Comment:
I tried in my VSCode and I don't even see any TS error here to disable
But I see `enforce: "post",` down below has error though
Are you sure `enforce: "post",` is working. I don't even know what it's doing.
Either way for be `pnpm -r build` works even if I remove this comment completely. The message in the comment doesn't give any value for the reader anyway in my opinion

Reply:
My IDE show the error here for some reason. Looks like it doesn't affect anything, so yeah, i removed comment there.